students/hyunsoo-jeon/user/views.py

```diff
@@ -0,0 +1,42 @@
+import json, re
+
+from django.views             import View
+from django.http              import JsonResponse
+from django.core.exceptions   import ValidationError
+
+from .models                  import User
+
+
+class SignupView(View):
+    def post(self, request):
+        data      = json.loads(request.body)
+        email_reg = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+        regex     = re.compile(email_reg)
+
+        try:
+            if not regex.match(data['email']):
+                return JsonResponse ({"message": "This email is not valid"}, status=400)
+
+            if len(data['password']) < 8:
+                 return JsonResponse ({"message": "This password is too short"}, status=400)
```

Comment:
- 리턴 메시지는 클라이언트가 보는 것이 아닌고 프론트엔드가 보는 메시지 입니다. 굳이 문장으로 안 하시고, 간단하게 표현해도 좋을 것 같습니다.
- 메시지 형식은 통일하는 것이 좋습니다. 다 대문자로 썼으면 대문자로만, _을 사용해서 단어을 이였다면 계속 _을 사용하는 것처럼요
```suggestion
                 return JsonResponse ({"message": "INVALID_PASSWORD"}, status=400)
```